operations/observability/mixins/workspace/rules/components/workspaces/alerts.libsonnet

```diff
@@ -134,6 +134,20 @@
               gitpod_ws_manager_workspace_phase_total{phase="PENDING", type="PREBUILD"} > 20
             |||,
           },
+          {
+            alert: 'GitpodWorkspaceTooLongTerminating',
+            labels: {
+              severity: 'warning',
```

Comment:
I was wondering if it should be `critical`.

Reply:
Good point, @utam0k ! Looking at warnings while on-call is smart.